src/services/itemLogin/schemas.ts

```diff
@@ -9,89 +10,100 @@ import { errorSchemaRef } from '../../schemas/global';
 import { accountSchemaRef } from '../account/schemas';
 import { itemSchemaRef } from '../item/schemas';
 
-export const credentialsSchemaRef = registerSchemaAsRef(
-  'itemLoginCredentials',
-  'Item Login Credentials',
-  Type.Object(
-    {
-      username: customType.Username(),
-      password: Type.Optional(
-        Type.String({ minLength: 3, maxLength: 50, pattern: '^\\S+( \\S+)*$' }),
-      ),
-    },
-    { additionalProperties: false },
-  ),
-);
-
-const itemLoginSchemaSchema = Type.Object(
+const itemLoginSchemaSchema = customType.StrictObject(
   {
     id: customType.UUID(),
-    type: customType.EnumString(Object.values(ItemLoginSchemaType)),
-    status: customType.EnumString(Object.values(ItemLoginSchemaStatus)),
+    type: customType.EnumString(Object.values(ItemLoginSchemaType), {
+      description: 'Defined which credentials are necessary to login.',
+    }),
+    status: customType.EnumString(Object.values(ItemLoginSchemaStatus), {
+      description:
+        'Whether the item login is enabled, freezed or disabled. An item login is never deleted, but disabled instead to prevent attached guests to be deleted.',
```

Comment:
```suggestion
        'Item login status, which can be enabled, frozen, or disabled. While never deleted, an item login can be disabled instead to prevent deleting associated guest accounts.',
```